Gemfile

```diff
@@ -79,3 +79,7 @@ group :test do
   gem 'capybara'
   gem 'selenium-webdriver'
 end
+
+gem "tailwindcss-rails", "~> 2.0"
+
+gem "foreman", "~> 0.87.2"
```

Comment:
[From their docs,](https://github.com/ddollar/foreman) `foreman` should not be in the Gemfile. You get foreman for free when you use `bin/dev` in your Rails app since Rails 7.

<img width="493" alt="Screenshot 2023-11-03 at 20 31 57" src="https://github.com/eargollo/soccer/assets/1158152/c9a4acb7-4c86-4f70-8e4f-62256d17b940">

Reply:
And, more on the `bin/dev` command that now comes with Rails 7 out of the box, since so many Rails apps used `foreman` for years:

https://railsnotes.xyz/blog/procfile-bin-dev-rails7#bindev-foreman-and-procfiledev--how-do-they-work-together

Reply:
I thought `bin/dev` was a thing of Tailwind. I was not using it until I installed Tailwind and read their docs.

Reply:
Fixed in #21

Reply:
`bin/dev` is basically a thin wrapper around `foreman`, since `foreman` is only run in dev environments and is used to run multiple processes to run the app.

In dev, you will need a process for the rails server at a minimum, and anything else will also need to be run in a separate process) (like Sidekiq, tailwind dev builds, etc.)

In production, these processes are run by the host machine, but in dev, you would need to start them all separately in separate terminal tabs or background processes. Foreman makes this easier, and `bin/dev` just wraps it.